src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -627,6 +627,13 @@ paths:
     post:
       operationId: ValidatorsUptime
       summary: Get Validators Uptime
+      description: |
+        Returns validators uptime data for time range limited by from_state_version and at_state_version.
```

Comment:
Use backticks whenever we reference some property, i.e. "from_state_version" => "`from_state_version`"